persistence/cosmosdb/transactions_information_CosmosDB_[,1].partial.md

```diff
@@ -0,0 +1,49 @@
+A custom [behavior](/nservicebus/pipeline/manipulate-with-behaviors.md) must be introduced to identify and insert the [`PartitionKey`](https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.documents.partitionkey?view=azure-dotnet) value into the pipeline context for use by storage operations that occur during the processing of a given message.
```

Comment:
Should we leave this in? Or just delete it and only keep the upgrade guide?

Reply:
I am deleting it